.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: check-yaml
+    -   id: check-added-large-files
+    -   id: check-executables-have-shebangs
+    -   id: check-merge-conflict
+    -   id: detect-private-key
```

Comment:
Are there any issues with something expected in our repo?

Reply:
Not particularly. But I guess it's a good thing to have, in case anyone accidentally tries to push a file containing private keys. Do you think we should remove this one too?

Reply:
These all seem like useful checks to have.